examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+"""
+Mask and Plot Remote Sensing Data with EarthPy
+==============================================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
```

Comment:
This header is pretty generic - do we need it?

Reply:
yes! good for SEO. it could be more specific i suppose.